docs/spl-docs/rex-command.md

```diff
@@ -0,0 +1,104 @@
+# rex
+
+## Description
+
+Use this command to either extract fields using regular expression named groups, or replace or substitute characters in a field using sed expressions.
+
+The rex command matches the value of the specified field against the unanchored regular expression and extracts the named groups into fields of the corresponding names.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+**rex** \
+**[field=\<field>]** \
+**\<regex-expression>**
+
+## Required Arguments
+
+### field
+
+**Syntax:** `field=<field>` \
+**Description:** The field that you want to extract information from.
+
+### regex-expression
+
+**Syntax:** `"<string>"` \
+**Description:** The PCRE regular expression that defines the information to match and extract from the specified field.
+
```

Comment:
nit: I think this has to be a quoted string, and it's fairly clear from your Syntax. But restating this in the Description could be useful to make it even more clear